langgenius/dify
# fix: Turn off SWR automatic revalidation when window is focused

# Description

[Issues#3048 Why do we always trigger many requests every time we focus on the page ?](https://github.com/langgenius/dify/issues/3084)

if using SWR to process data requests in React, set revalidateOnFocus=false in config is better.

Fixes # (issue)

## Type of Change


# How Has This Been Tested?

1. Open any page
2. Blur the page
3. Focus the page
4. See the network in browser's console

The network page of console is very refreshing


# Suggested Checklist:
